src/utilities/buildClientSchema.js

```diff
@@ -118,16 +119,16 @@ export function buildClientSchema(
 
   // Given a type reference in introspection, return the GraphQLType instance.
   // preferring cached instances before building new instances.
-  function getType(typeRef: IntrospectionTypeRef): GraphQLType {
+  function getType<T: IntrospectionTypeRef>(typeRef: T): GraphQLType {
```

Comment:
I don't think moving this to a type argument should be necessary since the `T` variable isn't used anywhere other than here.

Reply:
This is useful to end up with more-restrictive types: if I explicitly pass in a `IntrospectionOutputTypeRef` I could change this to return a `GraphQLOutputType` at some point in the future. Though for now you're probably right that this is unnecessary (and we may want to just make that a different function).

Also I didn't realize this was an internal-only-function so even more unnecessary :) Will change.